public/app/features/admin/AdminEditOrgPage.tsx

```diff
@@ -52,23 +58,31 @@ export const AdminEditOrgPage: FC<Props> = ({ match }) => {
     return await getBackendSrv().put('/api/orgs/' + orgId, { ...orgState.value, name });
   };
 
+  const renderMissingUserListRightsMessage = () => {
+    return (
+      <Alert severity="info" title="Missing permissions">
```

Comment:
```suggestion
      <Alert severity="info" title="Access denied">
```
nit: I'd say access denied message is more descriptive, consistent and clear with the effect.